tests/unit-tests/Providers/TenancyProviderTest.php

```diff
@@ -71,7 +72,7 @@ public function current_website_id_is_included_in_notification_job_payload()
     {
         Event::fake();
 
-        $user = $this->userStory()->create();
```

Comment:
I was so confused about this one @vpratfr, I almost thought I missed a new feature in Laravel 😊